default/python/tests/conftest.py

```diff
@@ -8,6 +8,11 @@
 
 this_dir = os.path.dirname(__file__)
 
+sys.path.append(this_dir)
 sys.path.append(os.path.join(this_dir, '..', '..', 'python', 'AI/'))
-sys.path.append(os.path.join(this_dir, '..', '..', 'python', 'interface_mock', 'result'))
 sys.path.append(os.path.join(this_dir, '..', '..', 'python'))
+
+# This import put module to memory, after that we can import code form AI module without problem,
```

Comment:
For better grammar and more thorough guidance I would suggest replacing these two lines with 
```
# Since the 'true' freeOrionAIInterface is not available during testing, this import loads the 'mock' freeOrionInterface
# module into memory.  For testing purposes the mock interface provides values for certain constants which various
# AI modules rely upon.  Since any later imports of the freeOrionInterface module will simply refer back the first-
# loaded module of the same name, code from the AI modules can then be imported and tested, provided that such
# AI code only relies upon constants from the freeOrionAIInterface (and specifically does not depend on any of the
# various dynamic galaxy-state queries provided by the 'true' freeOrionAIInterface).
```

At least, that better communicates my current understanding of this.